xla/service/cpu/onednn_matmul.cc

```diff
@@ -230,8 +236,49 @@ ABSL_ATTRIBUTE_NO_SANITIZE_MEMORY void __xla_cpu_runtime_OneDnnMatMul(
     // array.
     weights_md =
         memory::desc({input_md.get_dims().back(), output_md.get_dims().back()},
-                     weights_md.get_data_type(), memory::format_tag::ab);
+                     weights_md.get_data_type(), weight_format);
+  } else if (matmul_config.optimization_config().with_acl_backend()) {
```

Comment:
You can call `tsl::port::IsAarch64CPU()` from cpu_info.h instead.
https://github.com/openxla/xla/blob/29715b5b586204349a3dfbc6806f094133e8eff6/third_party/tsl/tsl/platform/cpu_info.h#L164

Reply:
Please also add a comment:
```
// TODO(penporn): Make this code work with AOT cross-compilation, e.g., using
// x86 CPU to compile for aarch64 CPU, by passing the target machine
// architecture as a function parameter.
```